oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -299,11 +300,11 @@ private String getProcessorID(String stepping, String model, String family, Stri
             if (checkLine.contains(marker) && checkLine.trim().startsWith("0x00000001")) {
                 String eax = "";
                 String edx = "";
-                for (String register : checkLine.split("\\s+")) {
+                for (String register : ParseUtil.whitespace.split(checkLine)) {
                     if (register.startsWith("eax=")) {
-                        eax = register.replace("eax=0x", "");
+                        eax = StringUtils.replace(register, "eax=0x", "");

```

Comment:
This looks like the primary performance improvement, replacing `String.replace()`'s inefficient regex implementation with `StringUtils.replace()` which uses a `StringBuffer`.   However, it would be even faster with a `StringBuilder` implementation.    If we want efficiency, can we just write our own instead of introducing an entirely new dependency simply to "delete this character sequence from our string" ?